app/src/main/java/campus/tech/kakao/map/data/network/PlaceResponse.kt

```diff
@@ -0,0 +1,7 @@
+package campus.tech.kakao.map.data.network
+
+import campus.tech.kakao.map.data.network.dto.Place
+
+data class PlaceResponse(
+    val documents: List<Place>,
+)
```

Comment:
모든 응답에는 @SerializedName 을 붙여주는 편이 조금 더 명확할 것 같아요!

Reply:
또한 응답이 만약 아래와 같이 null 로 내려온다면 해당 부분은 에러를 발생할 수 있습니다. nullable 처리를 해주시면 좋을 것 같아요!
```json
{
  "documents": null
}
```

```suggestion
data class PlaceResponse(
    val documents: List<Place>?,
)
```